### PR TITLE
Fix mgmt plane tests. Add tests for new features

### PR DESCRIPTION
**Pull request by yifanz0**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/RegressionTests/srp_preview.ps1 [hunk 1]
```diff
@@ -34,6 +34,119 @@ Describe "Management plan test - preview" {
 
         $Error.Count | should -be 0
     }
+
+    It "LCM TierTo Cold, TierToHot" {
```
Comment (blueww) [inline src/Storage/RegressionTests/srp_preview.ps1:38]: (Optional) We can set TierToHot, TierToCool together, to save the test time.

**Review by blueww — APPROVED** [review]
Seems the user identity cases are not included.
Will they in another PR?

**Comment by yifanz0** [comment]
@blueww The reason of the user identity case failure is that the key vaults didn't exist somehow. Thus creating the corresponding key vaults fixed the issue, and no code change is needed

File: src/Storage/RegressionTests/srp_preview.ps1 (hunk 1 above)
Comment (yifanz0) [reply src/Storage/RegressionTests/srp_preview.ps1:38]: Will combine them later. Was having some trouble creating premium blob storage account under region eastus2euap, which is needed if we want to test both TierToCold and TierToHot at the same time.

**Pull request merged**